src/rpm/builder.rs

```diff
@@ -762,7 +762,11 @@ impl RPMBuilder {
                 offset,
                 IndexData::Int32(vec![self.epoch]),
             ),
-            // @todo: write RPMTAG_RPMVERSION?
+            IndexEntry::new(
+                IndexTag::RPMTAG_RPMVERSION,
+                offset,
+                IndexData::StringTag(format!("rpm-rs {}", env!("CARGO_PKG_VERSION"))),
```

Comment:
What about tools like `cargo-generate-rpm`? Maybe it's worth to make it configurable, with `rpm-rs ...` being the default value?

Also, instead of `format!` you could use `concat!("rpm-rs ", env!("CARGO_PKG_VERSION"))`.

Reply:
It shouldn't be configurable because the part that actually matters for debugging purposes is knowing what version of this library the package was created by.  `cargo-generate-rpm` provides the metadata, but that's a very foolproof process compared to constructing a package from it.